src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -384,9 +384,15 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
             ErrorEvent += delegate (object o, EventArgs args)
             {
                 // NOTE: Exceptions leaked from this method may cause VS to crash, be careful
-
                 ResultEventArgs result = (ResultEventArgs)args;
-                _callback.OnError(result.Results.FindString("msg"));
+                string msgString = "msg";
+                // In lldb, the format is ^error,message=""
+                // In gdb/vsdbg it is ^error,msg=""
+                if (_launchOptions.DebuggerMIMode == MIMode.Lldb)
```

Comment:
Since this is a bug in LLDB that they might eventually fix, I might suggest instead Trying to fix 'msg', and if that property doesn't exist look for 'message'